### DISCUSSION TITLE
Kino der Toten (For realsies)

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#708] [Map Releases]
Hey guys. To be frank, I got a little tired of people asking for Kino der Toten to be added to the game when it was already kinda sorta ready-to-go and only needed a few hours of work left to be playable. I've decided to be even more generous than I already had been and collaborated with @LexiAshe to release it in a playable state here.

![fte-20240421011135-0](https://github.com/nzp-team/nzportable/assets/26030250/c4a91f5d-62be-4ce9-a688-821ef761c32d)

This is NOT a beautiful map. It's a gross mess, it was made over 9 years ago and left to rot because no one wanted to/was able to help rebuild it. But we've done a few things to make sure it runs relatively smoothly, and made sure it uses the new quality of life features when applicable. This includes an easter egg song, so have fun with that.

Feel free to submit patches to fix issues here for the download link to be updated and reflect those changes. I likely will not be maintaining this map myself. Sources are included. Enjoy!

# [DOWNLOAD](https://archive.org/download/kino_20251029/kino.zip)

**Comment by StickyPuddingg** [comment]
Thank you for another contribution that goes beyond what anyone could've hoped for!

I never had the pleasure of playing this map as a kid so I'm grateful to get a taste for it now.  

Also fixing something thats old because you feel like you have to is a mood, but fixing something that is 9 years old for a passion project is incredible, outstanding job regardless of if its perfect or not!

p.s i dont know if this is the right place to comment my appreciation and i apologize if it isnt. I'd have no quarrels if it has to be removed, thanks!

**Comment by LorenzoMorelli** [comment]
Works great on psvita. Thanks!

**Comment by sealteamaqua** [comment]
so... i downloaded, how precisely do i play it.

**Reply by SnugPugUnder** [reply]
go into the file, click maps, then click nzp.kinodertoten.txt

**Reply by SnugPugUnder** [reply]
drag it into the nzp.gay website

**Reply by Revnova** [reply]
> go into the file, click maps, then click nzp.kinodertoten.txt

i believe you're giving incorrect information. you open the zip. if the contents start with folders like **maps**, **gfx**, **models**, etc... then you drag and drop those folders into the nzp folder. it should merge.

**Reply by conbobcob** [reply]
i did the stuff but it just says a bunch of errors

**Comment by SnugPugUnder** [comment]
It’s a great map! Everything is perfect other than the waypoints. Zombies are getting stuck in corners. They are getting stuck in the room with the theater balcony. They spawn in the window but then they run into the corner for some reason. Please fix!

**Reply by DerpedCrusader** [reply]
A big fix is in the works at this very moment

**Reply by kronictripy420** [reply]
this is awesome on the switch amazing work my friend one thing is there a way to play this online with other ppl thanks in advanced

**Comment by conbobcob** [comment]
this is what happens when i drag the file into the website
![Screenshot 2024-07-16 8 09 28 PM](https://github.com/user-attachments/assets/82b08399-e40b-42ed-a750-172c856072f1)

**Comment by MoltenWolf85** [comment]
I only have access to the web version, and I cannot use any of these maps. How do I run them?

**Reply by MoltenWolf85** [reply]
It would also be better if you could respond sooner rather than later.

**Comment by Nye12g** [comment]
why is every map i download like town an kino so laggy? zombie spawns seem to need work and just causes game to become unplayable on town, but on kino it works much smoother but would be wayyyyyyyy bettter with all perks not just the og 5

**Comment by MoltenWolf85** [comment]
I don't know where to put this, but someone should make Ascension and/or Call of the Dead. Arguably two of the best maps of all time.

**Reply by katniny** [reply]
https://docs.nzp.gay/
and that someone can be you!

**Reply by MoltenWolf85** [reply]
I'm not good at that sort of thing.

**Reply by DerpedCrusader** [reply]
Neither was I until I learned! These things take honing a skill, and id say it's a pretty fun time learning, maybe give it a shot, starting small ofc

**Reply by MoltenWolf85** [reply]
I also only have a chromebook soooooo

**Comment by ghost** [comment]
Great map! Thank you for making this available, so happy to have this map. I keep getting a pathfinding failure message around round 5 as I turn on the power. Did I do something wrong when I downloaded the map or is this just an issue with the waypoints?

**Comment by ClixzyFN** [comment]
I lost a run cus of this invisible thing you can see a small texture at the bottom
![image](https://github.com/user-attachments/assets/586962a3-c8c6-416a-82c1-a8804ebb9aac)

**Comment by PrimiseRichtofen** [comment]
I'm a failure and cannot get it to work what do I do

**Comment by Useless3** [comment]
every time I try to open the map, it basically crashes and sends me back into the main menu

**Reply by SgtCookieBobble** [reply]
Bumping this as well - crashing on Vita.

**Comment by DerpedCrusader** [comment]
so, this was worked on a while ago, but from what i remember, this update fixes most if not all spawns, fixes most if not all waypoints. some lighting touches and some small prop/detail work. feel free to report bugs if you find them!

~~[Download](https://archive.org/download/kino_20251029/kino.zip)~~ download from above!

[source files](https://archive.org/download/kino_20251029/kino_source.zip)

**Reply by alexisyobanny** [reply]
Hay bug como que es molesto y es que en el mapa hay como banderitas de 🏳‍🌈 y el clásico gato arcoiris xd

**Reply by DerpedCrusader** [reply]
I think you installed the map incorrectly, because it sounds like you're missing model files

I just tested on a clean install and it ran fine, so make sure you're copying ALL the files

**Reply by alexisyobanny** [reply]
Ya no me sale el bug gracias :D

Aunque hay más detalles en el mapa que me gustaría que corrigieras y son las lámparas grandes están de color negro es como una imagen png pero no recortada también en el escenario del cine hay un bug que desaparecen las cosas como que de repente fallan las Texturas y modelos desaparecen y vuelven a aparecer no se como explicarlo pero espero que soluciones eso xd

Ah y un detallito mas y creo que se te olvidó poner los nuevos de Pascua se suponía que al disparar oh golpearlo con el cuchillo las tres piedras de meteoritos tenía que sonar la clásica canción pero lo ise y no sono la rola xd

**Reply by mabayram10** [reply]
Hi @DerpedCrusader , when you go to the pack a punch room zombies encircle the return point and when you return to the map you die instantly because all the zombies moved to the returnpoint. could you also fix that?

**Reply by xDAAREx** [reply]
the map seems to crash on nzp vita

**Reply by xDAAREx** [reply]
the crash on vita seems to solely be caused by the .bsp file, i’ve tried a fresh install of the game with a fresh install of the updated map, and then trying a fresh install with a fresh install of the non updated map, the non updated one still works, i slowly replaced the matching files and the only incompatibility is the .bsp from the update, i might look into fixing it myself

**Reply by DerpedCrusader** [reply]
updated to fix ~~crash on vita~~ apparently there's another issue with it now, a couple stuck spots, and add zones

**Reply by nonMat** [reply]
Kino with zones is just peak

**Reply by FleeingFawn** [reply]
i made this github account just to comment on this. fantastic fixxing up you did @DerpedCrusader 👍 playing on PPSSPP for android on a mid-teir phone. i only have two small issues with this: i get pathfinding errors when hugging the wall going up or down the stairs of the starting room. my second issue is performance issues in the start room. lots of lamps i notice, i think that might be contributing to some of the lag i get. neither of these issues are game-breaking thankfully just minor

otherwise i admire how well you dolled the place up. i especially appreciate the podium being added in the starting room so that i'm no longer running into just a hitbox. thank you very much for sharing this with us and i look forward to seeing the next update:)

**Reply by DerpedCrusader** [reply]
I forgot to add the max view distance to the compile since i updated to toolbox

I can do another compile that should help with performance in spawn/on the stage 

And what kind of waypoint errors? It it just showing up, or is it actively disrupting gameplay?

**Reply by FleeingFawn** [reply]
awesome! and yeah the errors just show up, they don't disrupt gameplay. i tried to replicate it for a screenshot but it appears to not be doing it anymore 🤷‍♀️

**Reply by lilydesigns06** [reply]
Map's broken on switch, just doesn't even load in and crashes the game.

**Comment by patenaudejordan-creator** [comment]
i have a problem i have nzp on psp and for some reason the shaders are really bright its like a dog round with brightness turned all the way up but worse.  and its like that all the time on every map. any clue how to fix this

**Comment by Richardman56** [comment]
Does anyone know how to get this map to work on psp? Every time I launch it the whole psp crashes

**Comment by tysonbulls64-hash** [comment]
works perfectly for 3 seconds on ps vita then goes to 5 fps and becomes unplayable like i can still move and kill zombies but im moving extremely slow

**Comment by guifarias31** [comment]
I am having issues on PS Vita
Without the @DerpedCrusader update, the game is really slow at like 1 FPS or so
And with the update, the map crashes:

execing nzp.rc
execing default.cfg
execing config.cfg
execing autoexec.cfg
1 demo(s) in loop
Music will be looped
WARNING: SDL_SetRelativeMouseMode(SDL_TRUE) failed.
]
Music will not be looped
WARNING: SDL_SetRelativeMouseMode(SDL_TRUE) failed.
Cvar_Set: variable r_maxrange not found
======SERVER ERROR in MBOX_ParseMB2File:
MBOX_ParseMB2File: Expected "nzp_mbox2" but got nzp_mbox260.0 -208.0

EDICT 358:
origin         '-200.0 -416.0  30.0'
angles         '  0.0 180.0   0.0'
classname      mystery_box
spawnflags       1.0
Host_Error: Program error
Shutting down SDL sound

**Comment by patenaudejordan-creator** [comment]
<img width="1280" height="720" alt="WIN_20260626_17_08_35_Pro" src="https://github.com/user-attachments/assets/3cf65ca0-f3f8-4a5c-9a8d-5d500e718712" />
how do i fix this

**Reply by DerpedCrusader** [reply]
This isn't that map, you should post this in the comments of the town map

**Comment by jh1662** [comment]
There is a bug where if 2 or more ppl go into the teleporter then it sometimes glitches out a player to be stuck and flicker between two different locations.

**Reply by M0THRAK** [reply]
This is not just an issue on Kino, but an issue with all maps that have teleporters. Also the safe limit from my experience is two people, as long as the person you're teleporting with is as far apart as possible.

Here's a link to the bug report on this issue:

[Bug report](https://github.com/nzp-team/nzportable/issues/1391)

**Comment by phatpappa** [comment]
I don't think the easter egg music works on 3DS. I interacted with 3 meteor chunks but nothing plays

**Comment by Fleeing-Fawn** [comment]
Hello! I'm unsure if this mod is kept up with, but I'd like to report a minor bug. I am playing on PC, Windows 11. When using the teleporter, all of the zombies gather at the teleporter exit. When my time is up in the projector room, I'm teleported back to the start room, where all of the zombies ambush and kill me pretty much instantly.

Otherwise everything else works great! Wonderful work you've accomplished here 🙂

**Comment by RatchetGAMES** [comment]
How do I install this map on the browser version of nzp? Usually with other maps you simply just change the zip to pk3 and drag it into the browser but this doesn't work with this map i have been trying to find a way with no luck any advice?

**Comment by SillyZombie115** [comment]
Really fun map. 🙂👍

A bit laggy on PSP, but still manageable.
I'm sad that theres no track for the Meteors, and leads to VRIL crash.
But, still a really fun map.

I also love using WaW weapons on Kino; it really paints a picture of how Kino would be if Treyarch didn't delay the map and have it put in WaW instead.
Interesting choice for the Kar98k on the wall as spawn weapon and not the Double Barrel.
I LOVE where you put the FG-42 wall-buy 😉 (iykyk).
And the BEST PART... No Nova Crawlers! 😍

Good map!

**Reply by MotoLegacy** [reply]
a custom 115 cover was included in the original download. Derped just forgot to include it in latest release.

If you get a crashes you should report them.

**Reply by SillyZombie115** [reply]
Oh?

**Reply by DerpedCrusader** [reply]
working on this right now

**Reply by DerpedCrusader** [reply]
okay, redownload from the link in original post and it'll be updated. also includes the loading/menu graphics

**Comment by phatpappa** [comment]
Zombies getting stuck in the room with the mp40 in the latest version of the map. Next to the table knocked over on the stairs. Also getting stuck at the front row of seats near the stage. I think they are also getting stuck in the changing room or whatever it is to the side of the stage.

**Reply by 0xLLUUKKEE** [reply]
also experiencing this issue on 3ds
